config/default_records/request_types/004_limber_high_throughput.yml

```diff
@@ -156,6 +156,21 @@ limber_heron_lthr:
     - Sanger_tailed_artic_v1_384
     - PCR amplicon tailed adapters 96
     - PCR amplicon tailed adapters 384
+limber_heron_lthr_v2:
+  <<: *limber_htp_library
+  name: Limber Heron LTHR V2
+  request_class_name: IlluminaHtp::Requests::HeronTailedRequest
+  for_multiplexing: true
+  target_purpose_name: LB Lib Pool Norm
```

Comment:
I think you may have mentioned this in stand-up, but that will still tie us to fixed-submissions, as we'll still be generating the Pool tube up-front. But I think you said you'd not got to that bit yet.